python/fastflow/common.py

```diff
@@ -0,0 +1,187 @@
+"""Common functionalities for the fastflow package."""
+
+from __future__ import annotations
+
+import dataclasses
+from collections.abc import Hashable, Mapping
+from collections.abc import Set as AbstractSet
+from enum import Enum
+from types import MappingProxyType
+from typing import Generic, Literal, TypeVar
+
+import networkx as nx
+
+# Vertex type
+V = TypeVar("V", bound=Hashable)
+
+# Plane-like
+P = TypeVar("P")
+
+
+class Plane(Enum):
+    """Measurement planes in MBQC."""
+
+    # DO NOT change the associated values!
+    XY = 0
+    YZ = 1
+    ZX = 2
```

Comment:
```suggestion
    XZ = 2
```
We define `XZ` not `ZX` planes (see `graphix.pauli.Plane`). I think it's best to have the same notations

Reply:
Please see here: https://github.com/TeamGraphix/fastflow/pull/3#issuecomment-2266979399. I intentionally changed the definition.

Reply:
Fixed in 3e36c72391b760c331fe32121aed703ab1b3a4d6.